src/site/stages/build/drupal/assets.js

```diff
@@ -2,6 +2,12 @@ const getDrupalClient = require('./api');
 const cheerio = require('cheerio');
 const { PUBLIC_URLS } = require('../../../constants/drupals');
 
+const PUBLIC_URLS_NO_SCHEME = {};
+Object.entries(PUBLIC_URLS).map(item => {
```

Comment:
What do you think about using `reduce` instead of `map` since we're not using the returned array?

Reply:
Will push up an change. :)